test/interactions/doubleClickInteractionTests.ts

```diff
@@ -1,35 +1,56 @@
 ///<reference path="../testReference.ts" />
 
 describe("Interactions", () => {
-  describe("DoubleClick", () => {
-    let SVG_WIDTH = 400;
-    let SVG_HEIGHT = 400;
+  describe("DoubleClick Interaction", () => {
 
-    describe("onDblClick generic callback", () => {
+    describe("Basic Usage", () => {
+      let userClickPoint: Plottable.Point;
```

Comment:
Hmm I see I think now.  Can we rename to like `clickedPoint` or `userClickedPoint`?  I think I prefer the former.